src/AsyncGenerator.yml

```diff
@@ -0,0 +1,249 @@
+﻿solution:
+  filePath: ..\..\..\src\NHibernate.sln #wants to resolve from the execuing path.
```

Comment:
`NUnitAsyncCountepartFinder` instead of `NUnitAsyncCounte`**r**`part`**s**`Finder`. Could these typo be fixed?